<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajouter</class>
 <widget class="QDialog" name="ajouter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>671</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>Ajouter les vols</string>
   </property>
   <widget class="QLabel" name="numunique">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>numéro_unique :</string>
    </property>
   </widget>
   <widget class="QLabel" name="destdepart">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>destination_départ : </string>
    </property>
   </widget>
   <widget class="QLabel" name="destarrive">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>destination_arrivée : </string>
    </property>
   </widget>
   <widget class="QLabel" name="datedepart">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>date_départ  : </string>
    </property>
   </widget>
   <widget class="QLabel" name="datedarrive">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>date_arrivée  : </string>
    </property>
   </widget>
   <widget class="QLabel" name="duree">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>dure_vol :</string>
    </property>
   </widget>
   <widget class="QLabel" name="nbpassager">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>nbr_passagers :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="valeur1">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="valeur2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="valeur3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="valeur4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="valeur5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="valeur6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="valeur7">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="validation">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>430</y>
      <width>193</width>
      <height>28</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../vols/ressources pour app qt/tunisair/60745.png</pixmap>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>validation</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>ajouter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>517</y>
    </hint>
    <hint type="destinationlabel">
     <x>761</x>
     <y>567</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
